src/js/components/banner.js

```diff
@@ -0,0 +1,16 @@
+var select = require('../utils/select');
+var addClass = require('../utils/add-class');
+var removeClass = require('../utils/remove-class');
+var dispatch = require('../utils/dispatch');
+
+var headers = select('.usa-banner-header');
+headers.forEach(function (header) {
+  var click = function (e) {
```

Comment:
OK, here's how to avoid breakage on IE8 (which I think is worthwhile):

    (event.preventDefault) ? event.preventDefault() : event.returnValue = false;

... which can be anywhere in the function, _if_ we _do_ want `preventDefault()` behaviour.